src/to-have-style.js

```diff
@@ -14,6 +14,14 @@ function getStyleDeclaration(document, css) {
   return styles
 }
 
+function isInvalidStyleDeclaration(name, value, computedStyle) {
+  return (
+    name &&
+    !value &&
+    !computedStyle[name] &&
+    !computedStyle.getPropertyValue(name)
+  )
+}
```

Comment:
nit: keep blank line between function declarations:

```suggestion
}

```

Reply:
Thanks for the suggestion, have you thought of adding a pre-commit hook to automatically apply the library's code style automatically?